internal/command/meta.go

```diff
@@ -736,6 +736,27 @@ func (m *Meta) showDiagnostics(vals ...interface{}) {
 	}
 }
 
+const (
+	// StatePersistIntervalEnvVar is the environment variable that can be set
+	// to control the interval at which Terraform persists state. The interval
+	// itself defaults to 20 seconds.
+	StatePersistIntervalEnvVar = "TF_STATE_PERSIST_INTERVAL"
```

Comment:
What if this mirrored the header configuration to call out the feature in question? TF_STATE_SNAPSHOT_INTERVAL

Reply:
I can't remember the exact conversation around this 😅, but I was under the impression we didn't want to use the term "snapshot" in a public context (even though the HTTP header uses the term).